packages/auth-sdk/user.tsx

```diff
@@ -29,8 +31,11 @@ export function useAuthenticatedUser() {
    return query;
 }
 
-async function fetchAuthenticatedUser(apiOrigin: string): Promise<User | null> {
-   const response = await fetch(`${apiOrigin}/api/2.0/user`, {
+async function fetchAuthenticatedUser(
+   apiClient: IFixitAPIClient,
+   apiOrigin: string
```

Comment:
Looks like apiOrigin isn't used. Any reason why we pass the client and origin in here instead of getting the client from inside fetchAuthenticatedUser and getting the origin inside the client?

Reply:
Just forgot to remove a now-unused argument.